iids/analyzer/views.py

```diff
@@ -3,12 +3,37 @@
 
 # Create your views here.
 #loading our trained model
-print("Keras model loading.......")
+print(" Model loading.......")
 model = load_model('attack_labe.hdf5') #after training #TODO
 print("Model loaded!!")
 
-    def request():
-        return HttpResponse("<h1>Intelligent Intrusion Detection System</h1>")
-         return HttpResponse("The predicted attacked label is {}".format(label)) #label will be imported after finishing dvevelopment of analyzer
-         
-         
\ No newline at end of file
+
+#class Ml_Algo and functions 
+
+class Random_Forest_Classifier():
+    def __init__(self):
+        default_path = "//"
+        self.model = load_model("")
+      
+    def preprocessing(self, input_data):
```

Comment:
Also instead of using defining function many times in differenet models, should we define it separately?

Reply:
well, it can be in a diff app altogether